src/client/components/base/Input.tsx

```diff
@@ -40,6 +41,7 @@ const Input: React.FC<InputProps> = ({
   autoCapitalize = 'sentences',
   autoCorrect = 'on',
   spellCheck = true,
+  otherInputProps = {},
```

Comment:
Originally I used the react idomatic code of `...props` and `<input {...props}` but that went wonky in the forms. For example we are setting name properties on the <Input> fields but since those aren't explicit props handled here, they don't go on the actual inputs. But with `...props` they were added and then alongside the `<CSRFForm>` formdata all the fields were twice in the actual post body.

I am not a fan of otherInputProps but I couldn't see a way a good alternative that didn't involve reviewing all <Input> usages across the code for cases like that, nor did I want to add explicit props for the various <input> fields.

If there are alternatives I didn't consider, please let me know!